src/lighteval/models/endpoints/endpoint_model.py

```diff
@@ -90,6 +90,27 @@ def from_path(cls, path: str) -> "ServerlessEndpointModelConfig":
 
 @dataclass
 class InferenceEndpointModelConfig:
+    """
+    This class is designed to manage and define settings for deploying inference endpoints in machine learning models. 
+
+    Attributes:
+    endpoint_name (str, optional):The name of the inference endpoint. 
```

Comment:
```suggestion
    endpoint_name (str, Optional): Inference endpoint name (to use at creation or reuse)

Reply:
For all the following, you should
- change the word order to simplify the text (the name of the model -> model name)
- remove initial "The" at the beginning of descriptions
- make sure the pattern is consistent: `key (type, optional, default: ""): Description` (remove optional/default as needed, but if there is a default you should in theory include optional). Btw, all of these have defaults which are often None

You should also check the post init function which will give you more pointers on the logic, it can be good to add to the doc which parameters are not compatible with one another for example